__tests__/SharedValue.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { View, Button } from 'react-native';
+import { render, fireEvent } from '@testing-library/react-native';
+import { useSharedValue } from '../src';
+
+const ViewTestComponent = () => {
+  const sv = useSharedValue(1);
+  return <View testID="view">{sv.value}</View>;
+};
+
+function TestComponentWithSV() {
+  const sv = useSharedValue(1);
+
+  return (
+    <View>
+      <View testID="view">{sv.value}</View>
+      <Button
+        testID="button"
+        title="run"
+        onPress={() => {
+          sv.value = sv.value + 1;
```

Comment:
Unrelated, do `sv++` and `sv += 1` work on shared values?

Reply:
Actually on numerical value, yes, but the generally shared value is immutable.

Reply:
I see.